public/app/plugins/panel/canvas/ConnectionSVG.tsx

```diff
@@ -19,11 +19,12 @@ interface ConnectionInfo {
   info: CanvasConnection;
 }
 
+let idCounter = 100;
```

Comment:
Any reason we are starting at 100 vs 0?

Reply:
committed Ryan's suggestion.. bigger numbers look better? I'll change it.

Reply:
no good reason --just aesthetics of the increment 🤷